doc/authors_emeritus.rst

```diff
@@ -27,7 +27,7 @@
 - Peter Prettenhofer
 - (Venkat) Raghav, Rajagopalan
 - Jacob Schreiber
-- Du Shiqiao
+- 杜世橋 Du Shiqiao
```

Comment:
If I recall well, I think that the rendering was not fine with Chinese characters. We need to check before merging.

Reply:
I checked and the rendering is fine